server/issue.go

```diff
@@ -80,9 +81,7 @@ func (s *Server) registerIssueRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to fetch issue list").SetInternal(err)
 		}
 
-		for _, issue := range issueList {
-			s.setTaskProgressForIssue(issue)
-		}
+		// We don't set task progress to reduce the size of response payload.
```

Comment:
The setTaskProgressForIssue seems to be lightweight since it's all in memory operation.

I would rather keep it unless we know it does impact performance.

Also even if we do want to ignore, I would just comment the code instead of removing it otherwise this comment is strange.

Reply:
SG.